src/libraries/analytics/analyticsEventTracking.ts

```diff
@@ -112,7 +112,8 @@ export enum BetterSocialEventTracking {
   ANONYMOUS_CHAT_SCREEN_ATTACHMENT_UPLOAD_FAILED = 'AnonChat-ChatScreen_ChatActions-Drawer_mediauploadfailed'
 }
 
-const ENABLE_TOAST = ENV === 'Dev';
+// const ENABLE_TOAST = ENV === 'Dev';
+const ENABLE_TOAST = false;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Hardcoding `ENABLE_TOAST` to `false` removes the flexibility of enabling/disabling toast based on the environment. If this change is intentional and you no longer need the feature toggling based on the environment, then it's fine. However, if you still want to maintain that flexibility, consider reverting this change.
```diff
- const ENABLE_TOAST = false;
+ const ENABLE_TOAST = ENV === 'Dev';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->